templates/experiment.html

```diff
@@ -19,7 +19,9 @@
 
 function disable(button) {
     const element = document.getElementById(button);
-    element.className = element.className.replace("primary", "inactive");
+    if (element !== null){
```

Comment:
Does this really solve the root cause of the problem? How can it happen that the button is null?